restcomm/restcomm.docs/sources-asciidoc/src/main/asciidoc/release-notes.adoc

```diff
@@ -0,0 +1,47 @@
+= Restcomm Release Notes
+
+These notes group both Community and Telscale changes in a single document.
+Changes marked with (R) sign, means those changes are introduced for the respective
```

Comment:
"(R)" makes me think "why did they choose (R)? what does it stand for?"

Reply:
how about ($) ? 

;)

Reply:
I changed the tag to **commercial** and provided a new section to explain the tags used in the release notes:
* **commercial** available only for RestcommOne product
* **security improvement** changes related to security

Later we can add more tags to be used in order to categorize and enhance changes